src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppEvent.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.sendletter.logging;
+
+public final class AppEvent {
```

Comment:
Is there a reason why the two constants can't be in the class itself where it is used?

Reply:
As insight class may grow with additional functionality in there, event with descriptions too - good point to keep utility class separate

Reply:
I think we should never code for requirements which are unknown.In future something changes you can create a utility class if required.

Reply:
this is like an enum, imho good to group it somehow